test/trelloBoardsApi/trello-boards.test.js

```diff
@@ -0,0 +1,167 @@
+const { expect } = require('chai');
+const BoardsApi = require('../../src/trello-api')
+const guard = require('../utils/guard')
+const factory = require('../../src/factory/trello-factory')
+
+// let api;
+// let unauthenticatedApi;
+
+describe(" Trello Boards API tests (/boards)", function () {
+  let api;
+  let unauthenticatedApi;
+
+  before(async function () {
+    api = new BoardsApi();
+    api.authenticate();
+     });
+
+    unauthenticatedApi = new BoardsApi();
+
+
+  afterEach(async () => {
+    await testCleanup()
+  })
+
+
+  describe("Trello Create Board (POST /boards)", function () {
+
+    it("Can create new board (POST /boards)", async function () {
+
+      const boardToBeCreated = {
+        "name": "Bill Shmo",
+        "gender": "Male",
+        "email": `${factory.qaPrefix}${Math.random().toString(36).slice(2)}@example.com`,
+        "status": "active"
+      };
+
+      let response = await api.createBoard(boardToBeCreated)
+
+      expect(response).to.have.property('id').that.is.a('number')
+    })
+
+
+    })
+
+  
+    it("Error is returned when creating a board and with missing email from the request (POST /board)", async function () {
```

Comment:
there's no such a property as email in trello board object: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-post